converter/internal/common/convert_targets.go

```diff
@@ -38,22 +34,22 @@ func (f ConvertTargets) RiverTokenize() []builder.Token {
 	}
 
 	for ix, targetMap := range f.Targets {
-		for key, target := range targetMap {
-			if key == "__address__" {
-				toks = append(toks, builder.Token{Tok: token.LBRACK})
-				toks = append(toks, builder.Token{Tok: token.LCURLY})
-				toks = append(toks, builder.Token{Tok: token.LITERAL, Lit: "\n"})
-				toks = append(toks, builder.Token{Tok: token.LITERAL, Lit: key})
-				toks = append(toks, builder.Token{Tok: token.ASSIGN})
-				toks = append(toks, builder.Token{Tok: token.STRING, Lit: `"` + target + `"`})
-				toks = append(toks, builder.Token{Tok: token.COMMA})
-				toks = append(toks, builder.Token{Tok: token.LITERAL, Lit: "\n"})
-				toks = append(toks, builder.Token{Tok: token.RCURLY})
-				toks = append(toks, builder.Token{Tok: token.RBRACK})
+		keyValMap := map[string]string{}
+		for key, val := range targetMap {
+			if key == "__expr__" {
```

Comment:
`__address__` -> `__expr__` ? This looks to be some special handling for conversion but maybe spelling that out a bit more?

Reply:
Will do, this is how we inject the string for other component exports during tokenize for this type.